packages/client/src/components/note.vue

```diff
@@ -452,6 +456,24 @@ function readPromo() {
 				}
 
 				> .content {
+					&.isLong {
+						> .showLess {
+							width: 100%;
+							margin-top: 1em;
+							position: sticky;
+							bottom: 1em;
+
+							> span {
+								display: inline-block;
+								background: var(--panel);
+								padding: 6px 10px;
+								font-size: 0.8em;
+								border-radius: 999px;
+								box-shadow: 0 2px 6px rgb(0 0 0 / 40%);
+								border: 1px solid var(--renote);
```

Comment:
I think you copied this from the "show more" button but that has a different background so the design needs to be a bit different here. I think you can use the CSS variable for background color here instead of adding a border. In my opinion the border does not look that great and can be replaced by making use of the [`spread-radius`](https://developer.mozilla.org/en-US/docs/Web/CSS/box-shadow#spread-radius).

I tried around a bit and I think this looks better:
```suggestion
								box-shadow: 0 0 7px 7px var(--bg);
```
![image](https://user-images.githubusercontent.com/20990607/165926193-d22aa827-de39-4b17-80df-16a9ce0a9607.png)

What do you think?

Reply:
Yes with that background color it's much more readable without border. (I have close to zero talent to UI design)

on light themes it's still a bit hard to read, but not painfully hard. At least for me. Tbf, the whole UI is harder to follow in light theme for me.
![image](https://user-images.githubusercontent.com/183191/165932969-18516694-6bdc-4f8e-b4d5-4895e8a465c8.png)